src/libraries/System.Net.Quic/ref/System.Net.Quic.csproj

```diff
@@ -1,9 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
-    <!-- Even though this library's contract isn't exposed in the shared framework, it is built as part of the shared framework build
-       (because the System.Net.Quic\src references this project) and hence need to define its dependencies. -->
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
Don't we need to remove the `RequiresPreviewFeaturesAttribute` when exposing the assembly in the shared framework?

Reply:
I'm not sure I fully understand, but we want to have S.N.Quic as PreviewFeature, since we want to be able to tweak the API for .NET 8. I hope the way I did it is correct.